src/components/HeroCard/HeroCard.js

```diff
@@ -39,66 +58,79 @@ export default class HeroCard extends Component {
                                             <h3>Health</h3>
-                                            <p>200</p>
+                                            <p>{health}</p>
                                         </div>
                                     </div>
                                     <div className='col-1-of-3'>
                                         <div className="main-stats">
                                             <h3>Armor</h3>
-                                            <p>0</p>
+                                            <p>{armor ? armor : 0}</p>
                                         </div>
                                     </div>
                                     <div className='col-1-of-3'>
                                         <div className="main-stats">
                                             <h3>Shield</h3>
-                                            <p>0</p>
+                                            <p>{shield}</p>
                                         </div>
                                     </div>
                                 </div>
-                                    <div className="main-statsbox__description">
-                                        <p>Hanzo’s versatile arrows can reveal his enemies or fragment to strike multiple targets. He can scale walls to fire his bow from on high, or summon a titanic spirit dragon.</p>
-                                    </div>
+                                    {/* <div className="main-statsbox__description">
+                                        <p className="main-statsbox__description-text">{description}</p>
+                                    </div> */}
                             </div>
                         </Column>
                     </Row>
                     </div>
                     <div className="margin-sides-small">
                         <div className="row">
-                            <CounterTabs tabClick={this.handleTabClick} heroID="1" currentTab={currentTab} />
-                            <CounterTabs tabClick={this.handleTabClick} heroID="2" currentTab={currentTab} />
-                            <CounterTabs tabClick={this.handleTabClick} heroID="3" currentTab={currentTab} />
+                            {counterOne &&<CounterTabs tabClick={this.handleTabClick}  
+                            currentTab={currentTab} 
+                            heroID={counterOne.id} counter={counterOne}/>}
+
+                            {counterTwo &&<CounterTabs tabClick={this.handleTabClick}  
+                            currentTab={currentTab} 
+                            heroID={counterTwo.id} counter={counterTwo}/>}
+
+                            {counterThree &&<CounterTabs tabClick={this.handleTabClick}  
+                            currentTab={currentTab} 
+                            heroID={counterThree.id} counter={counterThree}/>}
+
```

Comment:
Using an array called `counterHeroes` would allow:
```javascript
{
    counterHeroes.map((counter) => {
        const {id} = counter;
        return (<CounterTabs tabClick={this.handleTabClick} currentTab={currentTab} heroID={id} counter={counter} />);
    })
}
```

Reply:
Also, again, you're passing in the hero object for the counter hero, and then ALSO passing in the id that you pulled off that object. Think about if that's necessary redundancy.

Reply:
Whoops... Just realized you'll need a `key={id}` attribute on the `<CounterTabs>` object I posted above.